UsageExamples/OoplesIndicatorLibraryExample.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Alpaca.Markets;
+using OoplesFinance.StockIndicators.Models;
+using static OoplesFinance.StockIndicators.Calculations;
+
+namespace UsageExamples
+{
+    public class OoplesIndicatorLibraryExample
+    {
+        const string paperApiKey = "REPLACEME";
+        const string paperApiSecret = "REPLACEME";
+        const string symbol = "AAPL";
+
+        public async Task RunExample()
+        {
+            var startDate = new DateTime(2021, 01, 01);
+            var endDate = new DateTime(2021, 12, 31);
+
+            var client = Environments.Paper.GetAlpacaDataClient(new SecretKey(paperApiKey, paperApiSecret));
+            var bars = (await client.ListHistoricalBarsAsync(new HistoricalBarsRequest(symbol, startDate, endDate, BarTimeFrame.Day)).ConfigureAwait(false)).Items;
+            var stockData = new StockData(bars.Select(x => x.Open), bars.Select(x => x.High), bars.Select(x => x.Low), bars.Select(x => x.Close), bars.Select(x => x.Volume), bars.Select(x => x.TimeUtc));
+
+            // can do typical indicators such as using typical bollinger band settings
+            var results = stockData.CalculateBollingerBands();
```

Comment:
Extract a separate method for calculating and printing this indicator.

Reply:
I'm getting confused by this comment and the one below. I understand having a method to handle printing out the results of any indicator but the calculation is already done in one line so did you mean you want a method to handle the calculation process for all 3 examples I gave?